stellargraph/core/graph_networkx.py

```diff
@@ -772,6 +772,29 @@ def to_adjacency_matrix(self, nodes: Optional[Iterable] = None):
             self._graph, dtype="float32", weight=self._edge_weight_label, format="coo"
         )
 
+    def to_networkx(self):
+        # This is implemented generically to not be tied to the (outgoing) networkx format.
```

Comment:
What do you mean by this comment? Is this explaining why we don't just return `self._graph`?

Reply:
Yeah, maybe something like the following would be clearer:

```suggestion
        # This is not tied to the (to-be-removed) networkx-based StellarGraph, so it can easily be reused.
```

Reply:
I went with something similar but slightly different.